common/sgx/verifier.c

```diff
@@ -229,11 +229,17 @@ static oe_result_t _fill_with_known_claims(
     oe_result_t result = OE_UNEXPECTED;
     oe_report_t parsed_report = {0};
     oe_identity_t* id = &parsed_report.identity;
+    oe_parsed_tcb_info_t parsed_tcb_info = {0};
     const sgx_report_body_t* sgx_report_body = NULL;
+    sgx_quote_t* sgx_quote = NULL;
     size_t claims_index = 0;
     bool flag;
     oe_sgx_tcb_status_t tcb_status =
         oe_tcb_level_status_to_sgx_tcb_status(platform_tcb_level->status);
+    oe_tcb_info_tcb_level_t local_platform_tcb_level = *platform_tcb_level;
```

Comment:
Why doing this extra copy? Can't we just pass the pointer as-is to `oe_parse_tcb_info_json`?

Reply:
the call 'oe_parse_tcb_info_json' is modifying the status bit of the pointer passed in - https://github.com/openenclave/openenclave/blob/master/common/sgx/tcbinfo.c#L783

I didn't want it to change of the fields in the structure.

Reply:
It seems that the `platform_tcb_level` should be the out parameter of the 'oe_parse_tcb_info_json'. In that case, you should just parse a empty `oe_tcb_info_tcb_level_t`. Can you double check?

Reply:
Here it is said to be a [in] parameter - https://github.com/openenclave/openenclave/blob/master/common/sgx/tcbinfo.h#L128
but gets modified within the function. I'll double check offline if this can be different.